features_and_tools/cql.rst

```diff
@@ -0,0 +1,400 @@
+                        }
+                    ]
+                }
+
+Firely Server CQL Plugin 
+------------------------
+
+This chapter details setting up Firely Server CQL Plugin and provides you with an example to get some hands-on experience with CQL library with Postman tool. 
+ 
+* Getting Started 
+* CQL Plugin Configuration 
+* Using Postman 
+
+Getting Started
+^^^^^^^^^^^^^^^^
+
+* Install Firely Server using `Basic installation — Firely Server documentation <https://docs.fire.ly/projects/Firely-Server/en/latest/getting_started/basic_installation.html>`_
+
+    .. warning::
+        Make sure that the license contains CQL Plugin. 
+
+CQL Plugin Configuration in Firely Server 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Edit appsettings.instance.json:
+
+    1. Set Repository to ``SQL`` and add the database configuration 
+    2. Add ``$evaluate-measure`` to the ``SupportedInteractions.TypeLevelInteractions``
+    3. Ensure ``$everything`` is present in the ``SupportedInteractions.InstanceLevelInteractions``
+    4. Insert ``Vonk.Plugin.Cql`` to PipelineOptions.Branches[Path="/"].Include before any other plugin 
+    5. Uncomment ``Vonk.Plugin.PatientEverything`` from the same list PipelineOptions.Branches[Path="/"].Include. 
+
+* Build and run the Vonk.Server project from your IDE 
+
+Using Postman 
+^^^^^^^^^^^^^
+CQL Plugin in Firely Server uses the ``$evaluate-measure`` operation to execute a CQL measure on a resource.
+To verify the result of our CQL measure, we use Postman tool
+
+Create a postman collection
```

Comment:
Maybe better to point to a link where a ready made postman collection can be found?

Reply:
I see you do that already below, in that case maybe the part you describe here is not necessary? Unless, as you state below, you add extra explanation on each step of the Postman collection.

Reply:
I will check this with rest of the team, either switch the order in which this is displayed or map the explanation of the steps based on the collection.